spec/spec_helper.rb

```diff
@@ -1,4 +1,7 @@
 # frozen_string_literal: true
+require 'simplecov'
+SimpleCov.start
```

Comment:
OOC, are there any parts of our code where we don't want to measure coverage? (Things like `config` in a Rails app is a good example.)